game/games.go

```diff
@@ -19,8 +19,11 @@ func NewGames() Games {
 	return Games{seq: 0}
 }
 
-func (g *Games) AddGame(name string) {
-	g.games = append(g.games, NewGame(name))
+func (g *Games) AddGame(name string) int {
+	// If we move to a DB, we'll need to yank this
```

Comment:
If we don't move to a db all this will need to be mutex'd right?

Reply:
Oh yeah, good call. I think we'd just have to mutex the seq increment. I'm pretty sure there's a atomic increment in the atomic package we could use.